src/two_player_collision_example.cpp

```diff
@@ -211,73 +199,73 @@ TwoPlayerCollisionExample::TwoPlayerCollisionExample(
       "NominalHeadingP2");
   // p2_cost.AddStateCost(p2_nominal_orientation_cost);
   
-  // Stay in lanes.
+  // cost for deviating from the center of the lane (for both p1 and p2)
   const Polyline2 lane1(
```

Comment:
The naming for this (and other) lanes is not so great -- it was originally (in all the other examples) numbered to correspond to the player that used it, but here there is more than one "lane" per player sometimes so the numbering doesn't really make sense.

Reply:
I've updated this. For each player that a particular lane is relevant to, I iterate the lane label now. For example, the 3rd lane associated with p1 is labeled lane3_p1, or the 1st lane associated with both players is labeled lane1_p1p2.

Reply:
great!